db/seeds/jokes.rb

```diff
@@ -1,35 +1,35 @@
+puts Joke.create(lead: "This bouncy castle costs twice as much this year.", punchline: "That's what I call inflation!")
+puts Joke.create(lead: "Why can't you hear a pterodactyl use the bathroom?", punchline: "Because his pee is silent!")
+puts Joke.create(lead: "Want to hear a joke about construction?", punchline: "I'm still working on it!")
+puts Joke.create(lead: "I warned you not to eat that Play-doh spaghetti.", punchline: "It's an impasta!")
+puts Joke.create(lead: "Why do people think the shovel was such an important invention?", punchline: "Because it's ground-breaking!")
+puts Joke.create(lead: "A man told me he hadn't gone to the bathroom in years.", punchline: "I reckon he's full of crap!")
+puts Joke.create(lead: "You can't tell too many lightbulb jokes.", punchline: "You'll get burned out!")
+puts Joke.create(lead: "I don't tell that lasagna joke anymore.", punchline: "It was too cheesy!")
+puts Joke.create(lead: "When did your mother go to the dentist?", punchline: "Tooth hurt-y!")
+puts Joke.create(lead: "Did you hear about the guy who invented LifeSavers?", punchline: "They say he made a mint!")
+puts Joke.create(lead: "Chicken panini walks into a bar and orders a beer.", punchline: "Bartender says 'Sorry, we don't serve food here!'")
+puts Joke.create(lead: "Why did the tissue start dancing?", punchline: "It got a little boogie in it!")
+puts Joke.create(lead: "Two peanuts left for a party, but one didn't make it.", punchline: "It was assaulted!")
+puts Joke.create(lead: "Sorry, Junior... I got fired from the calendar factory today.", punchline: "They don't like it when you take days off.")
+puts Joke.create(lead: "I dreampt I was a muffler last night.", punchline: "I woke up exhausted!")
+puts Joke.create(lead: "I just saw a great show about beavers.", punchline: "Best dam show I'd ever seen!")
+puts Joke.create(lead: "Why can't your bike stay up by itself?", punchline: "It's two tired!")
+puts Joke.create(lead: "Don't trust atoms.", punchline: "They make up everything!")
+puts Joke.create(lead: "I used to hate facial hair...", punchline: "but then it grew on me!")
+puts Joke.create(lead: "What does an annoying pepper do?", punchline: "It gets jalapeño face!")
+puts Joke.create(lead: "Why won't your nose ever grow twelve inches long?", punchline: "Because then it would be a foot!")
+puts Joke.create(lead: "What do you get when you cross a snowman and a vampire?", punchline: "Frostbite!")
+puts Joke.create(lead: "Two satellites got married.", punchline: "The ceremony was boring, but the reception was incredible!")
+puts Joke.create(lead: "Did you hear about the explosion at the cheese factory?", punchline: "Nothing was left but de brie!")
+puts Joke.create(lead: "You know why crabs never give to charity?", punchline: "They're shellfish!")
+puts Joke.create(lead: "Why did the scarecrow win an award?", punchline: "He was out standing in his field!")
```

Comment:
Line is too long. [109/80]<br>Do not write to stdout. Use Rails' logger if you want to log.